src/Config/ConfigLoader.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Robo\Config;
+
+use Grasmash\YamlExpander\Expander;
+
+/**
+ * Load configuration files, and fill in any property values that
+ * need to be expanded.
+ */
+class ConfigLoader implements ConfigLoaderInterface
```

Comment:
In general, I think we should add type hints and param/return docs for public methods.